webpack.common.ts

```diff
@@ -46,6 +52,17 @@ module.exports = {
   },
   module: {
     rules: [
+      {
+        test: /\.m?js$/,
+        include: MONACO_DIR,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            presets: ['@babel/preset-env'],
+            plugins: ['@babel/plugin-transform-runtime'],
+          },
+        },
+      },
```

Comment:
Webpack plugin now uses ESM loading of the monaco-editor. This requires a separate loading rules if encountered errors (which we did).